### DISCUSSION TITLE
Paste Ascii Data into Keyboard

### DISCUSSION THREAD

**Discussion by cjmonica** [hneemann/Digital#1300] [Q&A]
Hi

I'm using this most excellent program Digital and would like to be able to Paste text data into the Keyboard.  It does not seem possible.  I'm using the Keyboard component in combination with the Terminal component. I want to paste (alla Cut/Paste) text (an ascii file) into the keyboard as if it is being typed in.  Is there any way to achieve this Keyboard paste behavior?
Thank You,
Charlie.

**Comment by hneemann** [comment]
You could load the file into a ROM and then clock the data into the terminal.

**Reply by cjmonica** [reply]
Hi H.  

Yes, that was my fallback option.  It would be best if the Keyboard component allowed for a paste capability.  I'm wanting the pasted text to appear as-if being typed into the Keyboard (pasted into the input stream).  I can make due.

Thanks for this MOST EXCELLENT DIGITAL program!
Charlie.
